qutebrowser/config/configtypes.py

```diff
@@ -798,6 +799,18 @@ class File(BaseType):
 
     typestr = 'file'
 
+    def transform(self, value):
+        if not value:
+            return None
+        value = os.path.expanduser(value)
+        value = os.path.expandvars(value)
+        if not os.path.isabs(value):
+            if standarddir.config() is not None:
+                abspath = os.path.join(standarddir.config(), value)
+                if os.path.isfile(abspath):
```

Comment:
I'm confused about this check - isn't the same thing already checked in `validate`? If so, no need to check it here again.